src/Avalonia.Controls/SplitView/SplitView.cs

```diff
@@ -131,19 +136,15 @@ public class SplitView : ContentControl
 
         private Panel? _pane;
         private IDisposable? _pointerDisposable;
+        private SplitViewTemplateSettings _templateSettings = null!;
```

Comment:
How `null!;` is valid C# I'll never understand :) Wouldn't it be better to just move it out of the constructor like:

```csharp
private SplitViewTemplateSettings _templateSettings = new SplitViewTemplateSettings();
```

Or do you have to trigger the change notification going through the setter for things to work correctly? Bindings are connected after the class is initialized so I wouldn't really expect it to be an issue.

Reply:
Fixed that. One of several things I don't like about nullable ref types, so I'm not even sure why I did that tbh